__tests__/cookies-page.test.js

```diff
@@ -0,0 +1,171 @@
+/* eslint-env jest */
+
+const { setupPage } = require('../lib/jest-utilities.js')
+const configPaths = require('../config/paths.json')
+const PORT = configPaths.testPort
+
+let page
+const baseUrl = 'http://localhost:' + PORT
+
+const cookiesPageSelector = '[data-module="app-cookies-page"]'
+
+beforeAll(async () => {
+  page = await setupPage()
+})
+
+afterAll(async () => {
+  await page.close()
+})
+
+describe('Cookies page', () => {
+  beforeEach(async () => {
+    await page.goto(`${baseUrl}/cookies`)
+  })
+
+  afterEach(async () => {
+    await page.deleteCookie({ name: 'design_system_cookies_policy' })
+    await page.setJavaScriptEnabled(true)
+  })
+
+  it('without JavaScript it has no visible inputs', async () => {
```

Comment:
Should we check that the whole form group is hidden?

Reply:
Good idea